### PR TITLE
Dixie Car Bomb Recipe

### PR DESCRIPTION
**Pull request by mattduggan**
@stevepescatore @mike-yesware

### CODE REVIEW

File: recipes/dixie_car_bomb.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mike-yesware) [inline recipes/dixie_car_bomb.md]: Newlines for effect?

**Comment by mike-yesware** [comment]
:joy: This is _so_ good.

To read that is. I dunno if I wanna ever drink it.

**Comment by stevepescatore** [comment]
Genius.

**Comment by mattduggan** [comment]
https://github.com/Yesware/cocktails/pull/5

**Pull request closed without merging**